.github/workflows/ci.yml

```diff
@@ -7,36 +7,48 @@ jobs:
     strategy:
       matrix:
         ruby:
-          - 2.7
-          - 3.0
-          - 3.1
+          - '2.7'
+          - '3.0'
+          - '3.1'
+          - '3.2'
         gemfile:
           - rails5.0
           - rails5.1
           - rails5.2
           - rails6.0
           - rails6.1
           - rails7.0
+          - rails7.1
         exclude:
-          - ruby: 3.0
+          - ruby: '3.0'
             gemfile: rails5.0
-          - ruby: 3.0
+          - ruby: '3.0'
             gemfile: rails5.1
-          - ruby: 3.0
+          - ruby: '3.0'
             gemfile: rails5.2
-          - ruby: 3.0
+          - ruby: '3.0'
             gemfile: rails6.0
-          - ruby: 3.0
+          - ruby: '3.0'
             gemfile: rails6.1
-          - ruby: 3.1
+          - ruby: '3.1'
             gemfile: rails5.0
-          - ruby: 3.1
+          - ruby: '3.1'
             gemfile: rails5.1
-          - ruby: 3.1
+          - ruby: '3.1'
             gemfile: rails5.2
-          - ruby: 3.1
+          - ruby: '3.1'
             gemfile: rails6.0
-          - ruby: 3.1
+          - ruby: '3.1'
+            gemfile: rails6.1
+          - ruby: '3.2'
+            gemfile: rails5.0
+          - ruby: '3.2'
+            gemfile: rails5.1
+          - ruby: '3.2'
+            gemfile: rails5.2
+          - ruby: '3.2'
+            gemfile: rails6.0
```

Comment:
I don’t think we want to exclude the combination of 3.2 and Rails 6.1. 
Probably switching to `include` would simplify the matrix?
I think we can drop at least 5.0 and 5.1 (and add Ruby 3.3, while we’re here).

Reply:
Should we drop tests for 5.0 and 5.1 or completely drop support for them?

Reply:
Completely: update the gemspec, remove the gemfiles, etc.